test/nextcloud/1.1.0/templates/backup-postgres-hook.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.ixChartContext.isUpgrade -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "pre-upgrade-hook2"
+  annotations:
+    "helm.sh/hook": pre-upgrade
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": hook-succeeded
+    rollme: {{ randAlphaNum 5 | quote }}
+spec:
+  template:
+    metadata:
+      name: "pre-upgrade-hook2"
+    spec:
+      restartPolicy: Never
+      serviceAccountName: "{{ template "nextcloud.serviceAccountName" . }}"
+      containers:
+      - name: {{ .Chart.Name }}-postgres-backup
+        image: "postgres:13.1"
```

Comment:
It is repeating itself now, and when doing postgres image upgrade we might change only one and end up with two different postgres images. Do we have an ability to define it as a variable somewhere and use it like `{{ .Chart.Name }}`?